### PR TITLE
Add docs for new repo env vars

### PR DESCRIPTION
**Pull request by cdupuis**

### CODE REVIEW

File: docs/authoring/container-skills.md [hunk 1]
```diff
@@ -128,6 +128,12 @@ The following environment variables are available inside the container:
     Storage bucket to write objects to
 -   `ATOMIST_GRAPHQL_ENDPOINT=<url for querying the GraphQL API>` - Url to the
     Atomist GraphQL endpoint
+-   `ATOMIST_REPOSITORY_SLUG` - Slug _owner/repo name_ of repository that
```
Comment (ddgenome) [inline docs/authoring/container-skills.md:131]:
```suggestion
-   `ATOMIST_REPOSITORY_SLUG` - The _owner/name_ slug of repository that 
```

File: docs/authoring/container-skills.md [hunk 2]
```diff
@@ -128,6 +128,12 @@ The following environment variables are available inside the container:
     Storage bucket to write objects to
 -   `ATOMIST_GRAPHQL_ENDPOINT=<url for querying the GraphQL API>` - Url to the
     Atomist GraphQL endpoint
+-   `ATOMIST_REPOSITORY_SLUG` - Slug _owner/repo name_ of repository that 
+    triggered the skill
+-   `ATOMIST_REPOSITORY` - Name of repository that triggered the skill
+-   `ATOMIST_REPOSITORY_OWNER` - Owner or organization of repository that
```
Comment (ddgenome) [inline docs/authoring/container-skills.md:134]:
```suggestion
-   `ATOMIST_REPOSITORY_OWNER` - Owner (user or organization) of repository that 
```

File: docs/authoring/container-skills.md [hunk 3]
```diff
@@ -128,6 +128,12 @@ The following environment variables are available inside the container:
     Storage bucket to write objects to
 -   `ATOMIST_GRAPHQL_ENDPOINT=<url for querying the GraphQL API>` - Url to the
     Atomist GraphQL endpoint
+-   `ATOMIST_REPOSITORY_SLUG` - Slug _owner/repo name_ of repository that 
+    triggered the skill
+-   `ATOMIST_REPOSITORY` - Name of repository that triggered the skill
+-   `ATOMIST_REPOSITORY_OWNER` - Owner or organization of repository that 
+    triggered this skill
```
Comment (ddgenome) [inline docs/authoring/container-skills.md:135]:
```suggestion
    triggered the skill
```

File: docs/authoring/container-skills.md [hunk 4]
```diff
@@ -128,6 +128,12 @@ The following environment variables are available inside the container:
...
+-   `ATOMIST_REF` - SHA, branch or tag of the event that triggered the skill
```
Comment (ddgenome) [inline docs/authoring/container-skills.md:136]: When would you ever get the branch? Wouldn't a push of commits or tags always be the trigger?

**Review by ddgenome — CHANGES_REQUESTED** [review]
A few suggestions and one question.

File: docs/authoring/container-skills.md (hunk 4 above)
Comment (cdupuis) [reply docs/authoring/container-skills.md:136]: We have triggers like Branch or DeletedBranch that purely trigger off of branch related events.

**Pull request merged**